src/js/experiments/kbLinks.js

```diff
@@ -0,0 +1,31 @@
+function renderExperimentalKnowledgeBaseLinks() {
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20790151733/variations
```

Comment:
would you mind just updating the link? https://app.optimizely.com/v2/projects/16812830475/experiments/21129180862/variations thanks!